bobtimus/src/main.rs

```diff
@@ -0,0 +1,17 @@
+#![deny(warnings)]
+use warp::Filter;
+
+#[tokio::main]
+async fn main() {
+    let cors = warp::cors().allow_any_origin();
```

Comment:
We shouldn't need cors if we serve the page from the same port.

Reply:
resolved due to proxy.